addons/website_slides/tests/test_slide_channel.py

```diff
@@ -274,6 +275,47 @@ def _get_completion_time_pdf(*args, **kwargs):
         self.assertEqual(13.37, round(slides_1.completion_time, 2))
         self.assertEqual(123.0, slides_2.completion_time)
 
+    def test_if_unlink_and_unpublish_channel_slides_sends_competition_email(self):
+        """When all content (slide.slide) are deleted from the course (slide.channel),
+        the course participants (slide.channel.partner) SHALL NOT receive completers email notification. """
+
+        with self.mock_mail_gateway():
```

Comment:
If it's here, we don't really know where it's used, and the whole method is indented.

We usually only use it where we use it, even if we repeat it:
```py
        self.assertA()
        with self.mock_mail_gateway():
            slide_published.with_user(self.user_manager).is_published = False
        self.assertB()
```

Reply:
Sure thing! Thanks for the snippet